eventmesh-common/src/test/java/org/apache/eventmesh/common/file/WatchFileManagerTest.java

```diff
@@ -53,6 +55,6 @@ public boolean support(FileChangeContext changeContext) {
         FileWriter fw = new FileWriter(file);
         properties.store(fw, "newAdd");
 
-        Thread.sleep(500);
+        ThreadUtils.sleep(500);
```

Comment:
@mxsm sleep(long timeout)  should add @Deprecated annotation.
repalce   sleep(long timeout)  with sleep(long timeout, TimeUnit timeUnit).